VideoWeb/VideoWeb/ClientApp/src/app/services/models/event-status.model.ts

```diff
@@ -0,0 +1,10 @@
+export class EventStatusModel {
+
```

Comment:
what does this model represent? What's the use case

Reply:
Could you reply to the question before closing a conversation going forward? Neither Ru or I could understand why you added this.

Reply:
I need to pass to the status component two parameters ,participant statuses (include judge) and judge statuses from another hearings, to calculate the requested status for judge by taking in account all hearing on the day, as it required in story. I have joined two parameters in one class.